src/roscompile/util.py

```diff
@@ -3,15 +3,12 @@
 HASH_PATTERN = re.compile('\n#+\n')
 
 def get_ignore_data_helper(basename):
-    #try:
-        fn = 'package://roscompile/data/' + basename + '.ignore'
-        lines = []
-        for s in get(fn).read().split('\n'):
```

Comment:
aha, what version are you using? I can look into it tonight maybe.

I by the way had the impression that I did need the try/catch block for some edge case, so I might reintroduce that too... But first I'll dig a bit deeper into it.